docs/sdk.rst

```diff
@@ -169,58 +169,73 @@ latter also works with the ``Executor`` class.
 Batching
 --------
 
-The SDK includes a batch interface to reduce the overheads of launching a function many times.
-To use this interface, you must first create a batch object and then pass that object
-to the ``batch_run`` function. ``batch_run`` is non-blocking and returns a list of task ids
-corresponding to the functions in the batch with the ordering preserved.
+The SDK includes a batch interface to reduce the overhead of launching a function many
+times.  To use this interface, first create a |Batch| object and then pass that
+object to the |batch_run()| method.  The |batch_run()| method only blocks for the
+duration of the API call; the tasks may not yet have been received by the endpoint when
+it returns.
 
 .. code-block:: python
 
+  def noisy_multiply(x, y, noise=1, offset=0) -> float:
+      import random
+      return x * y + random.uniform(offset - noise, offset + noise)
+
+  noisy_fn_id = gcc.register_function(noisy_multiply)
   batch = gcc.create_batch()
 
-  for x in range(0,5):
-    batch.add(x, endpoint_id=tutorial_endpoint, function_id=func_id)
+  for val in range(5):
+    batch.add(noisy_fn_id, val + 3, val)
```

Comment:
I don't think this works?  Isn't it 
batch.add(noisy_fn_id, args=(val + 3, val))
?  Or
batch.add(noisy_fn_id, (val+3, val))
?

Reply:
Och; that'll teach me (okay, no, it won't) to finish my thoughts at the far end of the day.  :facepalm: 

I'll fix that shortly.  Thanks for pointing it out!

... On the other hand, I don't see a reason it couldn't work, with a minor change to the signature.  I'll ponder that and perhaps create a PR after this one lands.